app/src/components/main/analytics/events/launchesPageEvents.js

```diff
@@ -300,6 +300,26 @@ export const LAUNCHES_PAGE_EVENTS = {
   REFINE_FILTERS_PANEL_EVENTS: {
     commonEvents: getRefineFiltersPanelEvents(LAUNCHES_PAGE),
   },
+  ADD_FILTER: {
+    category: LAUNCHES_PAGE,
+    action: 'Click on button "+Add filter"',
+    label: '',
+  },
+  refineBtnMore: (value) => ({
+    category: LAUNCHES_PAGE,
+    action: 'Select Parameter to Refine',
+    label: `${value}`,
+  }),
+  clickAttributes: (value) => ({
+    category: LAUNCHES_PAGE,
+    action: 'Click on icon Attributes',
+    label: `${value}`,
+  }),
```

Comment:
Please in case of using this func in many places like suite page, move it func into `testItemPages/actionEventsCreators` and use it like util.
Also according to the task not only launch and suite pages should be covered. Please add event on the test and step pages too.